ci/functional/test_main_prep_node.sh

```diff
@@ -38,9 +40,10 @@ function do_mail {
         return
     fi
     # shellcheck disable=SC2059
+    build_info = "BUILD_URL = $BUILD_URL$nl STAGE = $STAGE$nl$nl"
```

Comment:
(lint) Remove spaces around = to assign (or use [ ] to compare, or quote '=' if literal). [SC2283]
(lint) Possible misspelling: BUILD_URL may not be assigned. Did you mean BUID_URL? [SC2153]